.vscode/.ropeproject/config.py

```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
```

Comment:
Oh, not familiar with VS Code.. I'm guessing this keeps project settings synced across computers?

Reply:
Yes, I looked into this and I believe you're right. Was debating on whether to gitignore it or not.